source/OptimaJet.DWKit.StarterApplication/Repositories/UserRepository.cs

```diff
@@ -26,46 +26,47 @@ IDbContextResolver contextResolver
             this.CurrentUserContext = currentUserContext;
         }
 
-        protected IQueryable<User> DefaultGet()
-        {
-            return base.Get()
-                .Include(u => u.GroupMemberships)
-                    .ThenInclude(gm => gm.Group)
-                .Include(u => u.OrganizationMemberships)
-                    .ThenInclude(om => om.Organization);
-        }
         public override IQueryable<User> Get()
         {
-            if (OrganizationContext.SpecifiedOrganizationDoesNotExist) return Enumerable.Empty<User>().AsQueryable();
+            if (OrganizationContext.SpecifiedOrganizationDoesNotExist) {
+                return Enumerable.Empty<User>().AsQueryable();
+            }
+
+            var currentUser = GetByAuth0Id(CurrentUserContext.Auth0Id).Result;
+
+            var query = base.Get();
```

Comment:
I'm not sure if Entity Framework has an `or` query syntax, but if it did, that'd be a neat thing to add to the base query here to be shared with the two scenarios below to make sure the current-user is included in the results.